packages/api-core/src/site-scan/fetchers.ts

```diff
@@ -1,13 +1,6 @@
 import { wpcom } from '../wpcom-fetcher';
 import type { SiteScan, SiteScanHistory } from './types';
 
-export function enqueueSiteScan( siteId: number ): Promise< void > {
```

Comment:
Is this moving because it's a `POST` request? 🤔

Reply:
Mostly because of its purpose. It initiates a backend scan process (changes state) rather than just fetching data, so it belongs with mutators.

I’ve seen cases where fetchers use POST requests intentionally, but those are still considered fetchers 🙈